### DISCUSSION TITLE
zh/config/site

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#86] [Announcements]
# zh/config/site

版本 AList 的版本，只读。 Api地址 在v3.0.2及之后被弃用，已经移至site_url (./configuration.md#site-url) 你的 AList 的地址，比如 https://pan.nn.ci。这个地址会在程序中的某些地方使用，如果不设置这个字段，一些功能可能无法正常工作，比如 本地存储的缩略图; 开启web代理后的预...

https://alist.nn.ci/zh/config/site.html

**Comment by yubanmeiqin9048** [comment]
没有像V2版本那样的API了吗？[API Document](https://alist-doc.nn.ci/docs/api#publicpath)

**Comment by cancundeyingzi** [comment]
新版好难用,比如日志......2那多方便,3连个记录详细程度都没

**Reply by cancundeyingzi** [reply]
发错版块了...

**Comment by trhao99** [comment]
如何将我的配置从1个机器导出到另一个机器上

**Comment by happyostar** [comment]
### 设置选项找回

- 请问，设置-站点 页面中，那些带红色删除按钮的选项，我给点击删除了，如何找回？
- 使用群晖docker部署。

**Comment by AYOMITA** [comment]
我该如何使用站内的通知接口？我希望调用站内的通知来显示我的自定义内容
